Draft:Thermalize Lees Edwards LBM kernel

Yashas17 opened the pull request:
Fixes #

Description of changes:
- Add thermal fluctations to Lees Edwards LB kernel

Yashas17 commented on src/walberla_bridge/src/lattice_boltzmann/LBWalberlaImpl.hpp:
Since I am trying to thermalize Lees Edwards, I need to pass time-step to the kernel. The time-step getter and setter functions at the moment are called only if the collision model is the basic thermalized model. Since now even Lees Edwards will be thermalized and no unthermalized models remain, should I remove this check completely or add Lees Edwards collision model as an option as well. You can refer to line 535, 1777 and 1785 to check the current implementation?

Do you have any opinion on this @jngrad @itischler @RudolfWeeber?

jngrad replied:
If all collision models are thermalized, you no longer need to check the collision model type (and we can probably drop the `Thermalized` suffix from the regular kernels), but you still need to check for m_kT being zero.

Yashas17 committed:
Thermalized Lees-Edwards LB Kernels

- Changed generate_lb_kernels.py to thermalize LE.
- Added kT and seed parameters in all the LE functions in walberla
  bridge.
- Generated new kernels

Co-authored-by: RudolfWeeber <weeber@icp.uni-stuttgart.de>

RudolfWeeber committed:
Merge branch 'python' of ssh://github.com/espressomd/espresso into le_thermalized

RudolfWeeber committed:
LE Tracing

RudolfWeeber committed:
LE Tracing

RudolfWeeber committed:
Merge branch 'le_thermalized' of ssh://github.com/Yashas17/espresso into le_thermalized

Yashas17 committed:
Fix for LHS mismatch

Yashas17 committed:
Merge branch 'python' into le_thermalized
